devs/docs/es-backports.rst

```diff
@@ -141,26 +141,26 @@ should be crossed out as well.
 - [ ] 43ace5f80d7 Emit deprecation warnings when boosts are defined in mappings (#62623)
 - [ ] 9f5e95505bf Also abort ongoing file restores when snapshot restore is aborted (#62441) (#62607)
 - [ ] 06d5d360f92 Tidy up fillInStackTrace implementations (#62555)
-- [ ] 9bb7ce0b229 [7.x] Allocate new indices on "hot" or "content" tier depending on data stream inclusion (#62338) (#62557)
-- [ ] 91e23305295 Warn on badly-formed null values for date and IP field mappers (#62487)
+- [s] 9bb7ce0b229 [7.x] Allocate new indices on "hot" or "content" tier depending on data stream inclusion (#62338) (#62557)
+- [s] 91e23305295 Warn on badly-formed null values for date and IP field mappers (#62487)
 - [ ] e0a4a94985f Speed up merging when source is disabled. (#62443) (#62474)
 - [ ] 62dcc5b1ae1 Suppress stack in VersionConflictEngineException (#62433)
 - [ ] 5112c173194 Add WARN Logging on Slow Transport Message Handling (#62444) (#62521)
 - [ ] 14aec44cd86 Log if recovery affected by disconnect (#62437)
-- [ ] 24a24d050ab Implement fields fetch for runtime fields (backport of #61995) (#62416)
+- [s] 24a24d050ab Implement fields fetch for runtime fields (backport of #61995) (#62416)
```

Comment:
We don't have runtime fields and we skipped 
`0c8b4385777 Add support for runtime fields (#61776)`